languages/common-lisp/concepts/named-parameters/about.md

```diff
@@ -1 +1,16 @@
-TODO: add information on named-parameters concept
+In Common Lisp named parameters are called keyword parameters.
+
+Keyword parameters are designated by the `&key` lambda list keyword in a lambda list. Keyword parameters are not required, can have a default and also can specify a "supplied-p parameter" which will be "true" or "false" depending on whether an argument was provided for the parameter.
+
+In the arguments to a function the keyword parameters are specified by their "keyword name" which is, by default, a keyword symbol version of the parameter name (_i.e._ keyword parameter `name` has a keyword name of `:name`). It is possible to specify another keyword name by using a list of keyword name and parameter name instead of just the parameter-name:
```

Comment:
The sentence starting with "It is possible..." threw me for a bit of a loop the first time I read through it – I think it might b missing some plurals or something? It might be good to show an example of normal keyword parameters before this name-changed one as well :slightly_smiling_face: